### PR TITLE
Add the abilty to set WEB_SERVER_NAME/ALIAS and WEB_INCLUDES in ...

### PR DESCRIPTION
**Pull request by andytson-inviqa**
... php-apache images

### CODE REVIEW

File: php-apache/README.md [hunk 1]
```diff
@@ -85,12 +85,15 @@ The following variables are supported
 Variable | Description | Expected values | Default
 --- | --- | --- | ----
 WEB_HOST | The domain of the website | a domain | localhost
+WEB_SERVER_NAME | The domain matched by the virtual host in NGINX. Default is to match any hostname. | string |
```
Comment (kierenevans) [inline php-apache/README.md]: s/NGINX/Apache/

File: php-apache/README.md [hunk 2]
```diff
@@ -85,12 +85,15 @@ The following variables are supported
...
+WEB_SERVER_ALIAS | Additional domains matched by the virtual host in NGINX. | string/wildcard |
```
Comment (kierenevans) [inline php-apache/README.md]: s/NGINX/Apache/

**Comment by andytson-inviqa** [comment]
tested against #336 Spryker Apache images, and works as expected

**Comment by andytson-inviqa** [comment]
tho YVES_WEB_SERVER_NAME not being rendered :(

**Comment by andytson-inviqa** [comment]
appears to be a bug in that PR rather than this one

**Pull request merged**